jekyll/_cci2/runner-manually-upgrading-launch-agent.adoc

```diff
@@ -0,0 +1,141 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Upgrading the CircleCI Machine Runner on Cloud
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Find resources to manually upgrade an already installed CircleCI Machine Runner on linux, windows, or macOS.
+:icons: font
+:toc: macro:
+:toc-tittle:
+
+This page describes how to update the CircleCI **machine runner** on CircleCI cloud
+
+toc::[]
+
+[#check-current-version]
+== Checking current machine runner version
+
+The machine runner version can be checked by using the `--version` flag with the  binary. The default installation directory for the CircleCI machine runner binary is `/opt/circleci/circleci-launch-agent` on Linux as well as macOS, and `C:\Program Files\CircleCI\circleci-launch-agent.exe` on Windows.
+
+[#enable-auto-update]
+== Enabling auto update
+
+The most straightforward way to upgrade an existing CircleCI machine runner is to enable auto update in the configuration file and restart the runner instance. Auto update can be enabled by removing the `disable_auto_update` option from the machine runner configuration file, or by setting `runner.disable_auto_update` to false, then restarting machine runner instance either manually or by the process management used to manage the CircleCI machine runner agent. More details on configuring machine runner can be found in the xref:runner-config-reference[runner configuration reference] documentation.
+
+In some cases an environment variable may have been used to disable the auto update functionality. If the machine runner installation is not updated after enabling the auto update functionality in the configuration file, the environment variable `LAUNCH_AGENT_RUNNER_DISABLE_AUTO_UPDATE` should be set to have a value of `false`, or unset entirely, and the machine runner instance restarted again. 
+
+=== Linux 
+```shell
+sudo systemctl restart circleci.service
+```
+
+=== macOS
+```shell
+sudo launchctl stop '/Library/LaunchDaemons/com.circleci.runner.plist'
+sudo launchctl start '/Library/LaunchDaemons/com.circleci.runner.plist'
+```
+
+=== Windows
+```shell
+Stop-ScheduledTask -TaskName "CircleCI Launch Agent"
+Start ScheduledTask -TaskName "CircleCI Launch Agent"
+```
+
+[#manually-download-updated-binary]
+== Manually download the updated machine runner binary
+
```

Comment:
I think it would be great if we could lay out this section in a similar format as here: https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/runner-upgrading-on-server.adoc#upgrading-machine-runner-in-a-non-containerized-environment

e.g. 
```
== Manually update by downloading a new version of the machine runner binary

The process ... can be broken down into three steps... These steps are expanded upon in the sections below.

. Download a new version of the binary
. Replace the existing binary with the new version

=== 1. Download a new version of the binary
...
==== Download the binary on Linux or macOS
...
==== Download the binary on Windows
...

=== 2. Replace the existing binary with the new version
...
==== Replace the binary on Linux
...
==== Replace the binary on macOS
...
==== Replace the binary on Windows
```